closed/get_j9_source.sh

```diff
@@ -126,10 +126,7 @@ for i in "$@" ; do
 	esac
 done
 
-# clone OpenJ9 repos
-date '+[%F %T] Get OpenJ9 sources'
-START_TIME=$(date +%s)
-
+# Get OpenJ9, OMR sources
```

Comment:
This comment should be more general (internally there are other repos); please finish the sentence with a period.